docs/pages/includes/tctl-db-cli.mdx

```diff
@@ -0,0 +1,29 @@
+## tctl auth sign
+
+When invoked with a `--format=db` flag, produces a CA certificate, a client
+certificate and a private key file used for configuring Database Access with
+self-hosted database instances.
+
+```shell
+tctl auth sign --format=db --host=db.example.com --out=server --ttl=2190h
+```
+
+| Flag | Description |
+| - | - |
```

Comment:
Per https://github.com/gravitational/next/issues/305 - I'll convert these to HTML tables.